src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/FailoverGroupTests.cs

```diff
@@ -27,12 +27,118 @@ public FailoverGroupTests(ITestOutputHelper output)
             XunitTracingInterceptor.AddToContext(new XunitTracingInterceptor(output));
         }
 
-        // Currently the test runs too long to be marked as a check-in test.
-        [Fact(Skip = "Hydra TestFramework Issue, cannot login to authenticate in order to record the tests")]
+        [Fact]
         public void TestFailoverGroup()
         {
             RunPowerShellTest("Test-FailoverGroup");
         }
 
+        [Fact]
```

Comment:
@dlemMSFT you will need to add the `CheckIn` trait to each test for them to be run during the build. For example:

```cs
[Fact]
[Trait(Category.AcceptanceType, Category.CheckIn)]
public void TestCreateFailoverGroup_Named()
{
    RunPowerShellTest("Test-CreateFailoverGroup-Named");
}
```